src/devices/ServoMotor/ServoMotor.cs

```diff
@@ -0,0 +1,114 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Pwm;
+
+namespace Iot.Device.ServoMotor
+{
+    /// <summary>
+    /// Represents a rotary actuator or linear actuator that allows for precise control of angular or linear position.
+    /// </summary>
+    public class ServoMotor : IDisposable
```

Comment:
Should class be sealed? if not, and you expect people to derive from this, then we should probably implement the dispose pattern.

Reply:
Updated to sealed.  This should cover most scenarios for sweep and continuous types.